docker/klipper/Dockerfile

```diff
@@ -69,20 +72,16 @@ COPY config.simulavr .config
 RUN make 
 
 WORKDIR /opt
-RUN mkdir run
-RUN groupadd simulavr --gid 1000 \
- && useradd simulavr --uid 1000 --gid simulavr \
- && usermod simulavr --append --groups dialout
```

Comment:
This is necessary for simulavr to be able to create a serial device and cannot be omitted.